src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1176,6 +1176,12 @@ begin
   exact h.elim (λ q, hp q.to_path),
 end
 
+protected lemma adj.reachable {u v : V} (h : G.adj u v) :
+  G.reachable u v := ⟨walk.cons h walk.nil⟩
```

Comment:
```suggestion
  G.reachable u v := ⟨walk.nil.cons h⟩
```

Reply:
For this, I prefer not to use dot notation since putting the rest of a walk first is backwards, at least the way I look at walks.

What do you think about also including `adj.to_walk`?
```
protected def adj.to_walk {u v : V} (h : G.adj u v) :
  G.walk u v := walk.cons h walk.nil

protected lemma adj.reachable {u v : V} (h : G.adj u v) :
  G.reachable u v := ⟨h.to_walk⟩
```

Reply:
`adj.to_walk` sounds good! But no need to protect it, unless you name it `adj.walk`.